Bio/Align/Applications/_Muscle.py

```diff
@@ -72,275 +75,353 @@ def __init__(self, cmd="muscle", **kwargs):
                     filename=True,
                     equate=False),
-            # loga            File name            None.              Log file name (append
-            #                                                        to existing file).
+            # loga            File name            None.    Log file name
+            #                                              (append to existing
+            #                                              file).
             _Option(["-loga", "loga"],
                     "Log file name (append to existing file)",
                     filename=True,
                     equate=False),
-            # maxdiagbreak    Integer              1                  Maximum distance
-            #                                                        between two diagonals
-            #                                                        that allows them to
-            #                                                        merge into one
-            #                                                        diagonal.
-            _Option(["-maxdiagbreak", "maxdiagbreak"],
+            # matrix          File name            None.    File name for
+            #                                              substitution matrix
+            #                                              in NCBI or WU-BLAST
+            #                                              format. If you
+            #                                              specify your own
+            #                                              matrix, you should
+            #                                              also specify:
+            #                                                -gapopen <g>
+            #                                                -gapextend <e>
+            #                                                -center 0.0
+            _Option(["-matrix", "matrix"],
+                    "path to NCBI or WU-BLAST format protein substitution "
+                    "matrix - also set -gapopen, -gapextend and -center",
+                    filename=True,
+                    equate=False),
+            # diagbreak    Integer              1           Maximum distance
+            #                                              between two
+            #                                              diagonals that
+            #                                              allows them to
+            #                                              merge into one
+            #                                              diagonal.
+            _Option(["-diagbreak", "diagbreak"],
+                    "Maximum distance between two diagonals that allows "
+                    "them to merge into one diagonal",
+                    checker_function=lambda x: isinstance(x, int),
+                    equate=False),
+            _Option(["-maxdiagbreak", "maxdiagbreak"],      # deprecated 3.8
```

Comment:
I would include something like this in the docstring for this attribute: ``"Deprecated in v3.8, use -diagbreak instead."``